src/licensedcode/data/rules/unknown-license-reference_158.RULE

```diff
@@ -6,4 +6,4 @@ ignorable_urls:
     - https://go.microsoft.com/fwlink/?linkid=529443
 ---
 
-https://go.microsoft.com/fwlink/?linkid=529443
\ No newline at end of file
+https://dotnet.microsoft.com/en-us/dotnet_library_license.htm
```

Comment:
You've misunderstood entirely what we want to do here.

We don't want to change the rule text, we want to change the license_expression for this rule to `ms-net-library-2019-06` as this is a license we now have in licenseDB and so this is not an unknown license reference.

Note that we cannot just change a license-expression by replacing, we have to deprecate this rule and create a new rule, see the comment in https://github.com/nexB/scancode-toolkit/pull/3744#discussion_r1601669862 for example. As this is reference data and we want to preserve these rule names and links to these.